tests/platform_tests/fwutil/test_fwutil.py

```diff
@@ -0,0 +1,119 @@
+import re
+import pytest
+import os
+import json
+
+from fwutil_common import call_fwutil, show_firmware, upload_platform
+
+DEVICES_PATH="/usr/share/sonic/device"
+
+def test_fwutil_show(duthost):
+    # Test that command returns without error
+    assert show_firmware(duthost)
+
+def test_fwutil_install_file(duthost, localhost, pdu_controller, fw_pkg, random_component):
+    assert call_fwutil(duthost, 
+            localhost, 
+            pdu_controller, 
+            fw_pkg, 
+            component=random_component, 
+            basepath=os.path.join(DEVICES_PATH, duthost.facts['platform']))
+
+def test_fwutil_install_url(duthost, localhost, pdu_controller, fw_pkg, random_component, host_firmware):
+    assert call_fwutil(duthost,
+            localhost,
+            pdu_controller,
+            fw_pkg, 
+            component=random_component, 
+            basepath=host_firmware)
+
+def test_fwutil_install_bad_name(duthost, fw_pkg):
+    out = duthost.command("fwutil install chassis component BAD fw BAD.pkg", module_ignore_errors=True)
+    found = False
+    pattern = re.compile(r'.*Invalid value for "<component_name>"*.')
+    for line in out['stderr_lines']:
+        if pattern.match(line):
+            found = True
+            break
+    assert found
+
+def test_fwutil_install_bad_path(duthost, fw_pkg, random_component):
+    # Test fwutil with bad filepath to firmware
+    out = duthost.command("fwutil install chassis component {} fw BAD.pkg".format(random_component), module_ignore_errors=True)
+    found = False
+    pattern = re.compile(r'.*Error: Invalid value for "<fw_path>"*.')
+    for line in out['stderr_lines']:
```

Comment:
@alexrallen could it be part of `stdout`?

Reply:
Its in stderr, this test is currently passing on regression so I am quite certain here. However I will say it is definitely inconsistent with the error messages thrown by `fwutil update` which are in `stdout`